docs/meshcloud.project-metering.md

```diff
@@ -75,10 +75,19 @@ This export will contain the line items (see above) of all the chargeback statem
 The line item data is suitable for feeding into chargeback processing, e.g. importing it to an ERP System to transfer
 budgets between cost centers.
 
-Chargeback Statements also contain billing information per line item. Your Cloud Foundation team can [configure](meshstack.billing-configuration.md)
+For customers who export chargeback statements on a monthly basis, the following recommendation applies:
+Chargeback statements of the previous month should be processed no earlier than the `finalizeReportsAfterDays` setting
```

Comment:
d: sounds good to me. I just think we also should have it at another place. This one here is about CSV export and it makes sense to mention it here. But the other way and the preferred way to get the CBs is via the API. So we should either mention it additionally at a more central place, if it exists in these docs, or we could add it to the API docs of the meshChargeback API.